### ISSUE TITLE
Control Lock,Doors and Windows

### ISSUE THREAD

**Issue by seydx** [SeydX/homebridge-mercedesme#2]
The **BYOCARS API** offered from Mercedes allows only polling information about your car. At the moment it ist not possible to controll stuff like windows, doors or lock.

If in the future the possibility controlling these becomes possible, it would be better to change the services for doors and windows from **Service.ContactSensor** to

1. **Service.Door** for the doors to allow controlling them via HomeKit/Homebridge

2. **Service.Window** for the windows to allow controlling them via HomeKit/Homebridge

I have already contacted Mercedes in this regard and am waiting for a feedback

**Comment by NGDM** [comment]
I see they have a experimental API called [*Connected Vehicle*](https://developer.mercedes-benz.com/products/connected_vehicle)

Apparently this allows for door lock/unlock. 
Have you already analysed this API?

**Comment by SeydX** [comment]
@NGDM 

yes, the connected vehicle api only offers working with a simulation car. Not with real one.

**Comment by NGDM** [comment]
I see, let's hope they make it available to personal cars soon!
